ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -267,6 +267,19 @@ void OwInterface::initialize()
 {
   LanderInterface::initialize();
 
+  std::string fault_hierarchy_path;
+  bool verbose_flag = false;
+  if(m_genericNodeHandle->getParam("/verbose_fault_hierarchy_flag", verbose_flag)){
+    verbose_flag = verbose_flag;
+  }
+  if(m_genericNodeHandle->getParam("/fault_hierarchy_file", fault_hierarchy_path)){
+    m_fault_hierarchy = std::make_unique<FaultHierarchy>(fault_hierarchy_path, verbose_flag);
+  }
+  else{
+    ROS_ERROR("COULD NOT FIND THE FAULT HIERARCHY FILE");
+    m_fault_hierarchy = std::make_unique<FaultHierarchy>("", verbose_flag);
```

Comment:
Can FaultHeirarchy operate without a file, or does this just cause a bug downstream?